decidim-core/lib/decidim/faker/localized.rb

```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+require "faker"
+
+module Decidim
+  module Faker
+    # A Custom Faker wrapper so we can easily generate fake data for each
+    # locale in localized attributes.
+    class Localized
```

Comment:
You implementation reminds me of an M, man. Look at this indentation level :trollface: